Oswaldo Vazquez Aparicio/30 Agosto/hackerrank/JavaArrayList.java

```diff
@@ -59,7 +59,7 @@ private static void performQuery(int lineNumber, int position) {
 
         if (lineNumber <= linesNumber) {
 
-            ArrayList<Integer> line = table.get(lineNumber - 1);
+            List<Integer> line = table.get(lineNumber - 1);
```

Comment:
When you variable is a Collection, it is good practice put name in singular and them the the type, example: tableListList, userSet, etc...